src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,115 @@
+package duke;
+
+import java.util.List;
+import java.util.Scanner;
+
+public class Duke {
+
+  private Storage storage;
+  private TaskList taskArray;
+  private Ui ui;
+
+  public Duke(String filePath) {
+    this.ui = new Ui();
+    this.storage = new Storage(filePath);
+    List<String> taskArrayAsString = storage.load();
+    this.taskArray = new TaskList(taskArrayAsString);
+  }
+
+  private static final List<String> GREETING = List.of("Hello! I'm Fergus' Chatbot!", "What can I do for you?");
+  private static final String FAREWELL = "Bye. Hope to see you again soon!";
+  private static final String ERROR_UNKNOWN_COMMAND = "☹ OOPS!!! I'm sorry, but I don't know what that means :-(";
+
+  private static final String COMMAND_BYE = "bye";
+  private static final String COMMAND_LIST = "list";
+  private static final String COMMAND_DONE = "done";
+  protected static final String COMMAND_TODO = "todo";
+  protected static final String COMMAND_EVENT = "event";
+  protected static final String COMMAND_DEADLINE = "deadline";
+  private static final String COMMAND_DELETE = "delete";
+  private static final String COMMAND_SAVE = "save";
+  private static final String COMMAND_FIND = "find";
+  private static final String COMMAND_RESET = "reset";
+
+  /**
+   * Handles the user commands using a switch.
+   */
+  public void handleCommands() {
+    Scanner scanner = new Scanner(System.in);
+    boolean hasCommands = true;
+
+    while (hasCommands) {
+      String commandLine = scanner.nextLine();
+      String[] commands = commandLine.split(" ");
+      String command = commands[0];
+      switch (command) {
+        case COMMAND_BYE:
+          {
+            scanner.close();
+            hasCommands = false;
+            break;
+          }
+        case COMMAND_LIST:
+          {
+            this.taskArray.handleList();
+            break;
+          }
+        case COMMAND_EVENT:
+          {
+            this.taskArray.handleAddEvent(commands);
+            break;
+          }
+        case COMMAND_DEADLINE:
+          {
+            this.taskArray.handleAddDeadline(commands);
+            break;
+          }
+        case COMMAND_TODO:
+          {
+            this.taskArray.handleAddToDo(commands);
+            break;
+          }
+        case COMMAND_DONE:
+          {
+            int taskIndex = Integer.parseInt(commands[1]);
+            this.taskArray.handleDone(taskIndex);
+            break;
+          }
+        case COMMAND_DELETE:
+          {
+            int taskIndex = Integer.parseInt(commands[1]);
+            this.taskArray.handleDelete(taskIndex);
+            break;
+          }
+        case COMMAND_SAVE:
+          {
+            this.taskArray.handleSave(this.storage);
+            break;
+          }
+        case COMMAND_FIND:
+          this.taskArray.handleFind(commands[1]);
+          break;
+        case COMMAND_RESET:
+          this.taskArray.handleReset();
+          break;
+        default:
```

Comment:
Do you think switch and case should have the same indentation level to conform better with our styleguide?